utils/generate_js_fixtures.py

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env python3
+
+import subprocess
+from pathlib import Path
+
+SPHINX_ROOT = Path(__file__).resolve().parents[1]
```

Comment:
```suggestion
SPHINX_ROOT = Path(__file__).parent.parent.resolve()
```

(I think it's the correct one, but check it with a print).

Reply:
That does work, yep - it's less consistent with other scripts in the same directory, so I'm unsure whether to change it.

Reply:
I updated my suggestion since it's better to first resolve and do the parent.parent (in case you have symlinks). It's maybe less consistent but it's clearer. I mean, you need to remember that your parents are in the reverse order whereas with .parent.parent it's clearer to me.